### PR TITLE
DAOS-10863 control: Log warning for single access point (#9395)

### PR DESCRIPTION
**Pull request by kjacque**
While using a single access point is allowed, it provides no
redundancy. This log message provides a warning for the admin.

Signed-off-by: Kris Jacque <kristin.jacque@intel.com>

### CODE REVIEW

**Review by knard-intel — DISMISSED** [review]
OK for me.
From what I understand, the test failing is not related: probably a CI transient error.

File: src/control/server/config/server.go [hunk 1]
```diff
@@ -460,6 +460,9 @@ func (cfg *Server) Validate(log logging.Logger, hugePageSize int) (err error) {
 		return FaultConfigBadAccessPoints
 	case len(cfg.AccessPoints)%2 == 0:
 		return FaultConfigEvenAccessPoints
+	case len(cfg.AccessPoints) == 1:
+		log.Infof("WARNING: Configuration includes only one access point. This is provides no redundancy " +
```
Comment (mchaarawi) [inline src/control/server/config/server.go:464]:
```suggestion
		log.Infof("WARNING: Configuration includes only one access point. This provides no redundancy " +
```

Comment (kjacque) [reply src/control/server/config/server.go:464]: Oops. Good catch.

**Review by knard-intel — APPROVED** [review]

**Comment by kjacque** [comment]
The test failure was an incidence of DAOS-10301. I've restarted hardware testing.

**Review by tanabarr — APPROVED** [review]

**Pull request merged**